### PR TITLE
Improve steps wrapping behaviour

### PR DESCRIPTION
**Pull request by peteryates**
There was a minor discrepancy in the way Firefox and Chrome/Safari rendered the steps; Firefox was wrapping (reverting to `word-break: break-word` from the parent elmeent) beyond a certain point and Chrome and Safari weren't.

| Before | After |
| ------- | ------- |
| Screenshot from 2020-11-18 15-39-54 |  Screenshot from 2020-11-18 15-40-00 |

### CODE REVIEW

**Review by MylesJarvis — APPROVED** [review]

**Review by MylesJarvis — APPROVED** [review]
Trying to get this approval to work - hoping this does it? Change looks fine and we've discussed already.

**Comment by peteryates** [comment]
It worked fine @MylesJarvis, I can't merge until another reviewer with write access approves them too.

**Review by ethax-ross — APPROVED** [review]

**Pull request merged**
